File: VB/T306403/CustomColorEdit/CustomColorEditViewInfo.vb
Imports DevExpress.XtraEditors.Repository
Imports DevExpress.XtraEditors.ViewInfo
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace T306403.CustomColorEdit
    Public Class CustomColorEditViewInfo
        Inherits ColorEditViewInfo

        Public Sub New(ByVal item As RepositoryItem)
            MyBase.New(item)
        End Sub
    End Class
End Namespace

File: CS/T306403/CustomColorEdit/CustomColorEditViewInfo.cs
using DevExpress.XtraEditors.Repository;
using DevExpress.XtraEditors.ViewInfo;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace T306403.CustomColorEdit {
    public class CustomColorEditViewInfo : ColorEditViewInfo {
        public CustomColorEditViewInfo(RepositoryItem item)
            : base(item) {
        }
    }
}
